app/components/anchor/text_field_component.rb

```diff
@@ -15,16 +15,39 @@ class TextFieldComponent < Component
       [&[readonly]]:shadow-none
     ).freeze
 
-    def initialize(form_builder:, attribute:, **options)
+    ICON_WRAPPER_CLASSES = %w(
+      absolute
+      flex
+      inset-y-0
+      items-center
+      pointer-events-none
+    ).freeze
+
+    def initialize(form_builder:, attribute:, icon_left: nil, icon_right: nil)
```

Comment:
Let’s follow the naming convention from our Button component:

```suggestion
    def initialize(form_builder:, attribute:, starting_icon: nil, ending_icon: nil)
```

Reply:
Makes sense.